extensions/cornerstone-dicom-sr/src/components/OHIFCornerstoneSRMeasurementViewport.tsx

```diff
@@ -73,28 +58,6 @@ function OHIFCornerstoneSRMeasurementViewport(props: withAppTypes) {
     sendTrackedMeasurementsEvent = tracked?.[1];
   }
 
-  if (!sendTrackedMeasurementsEvent) {
```

Comment:
I don't understand why `sendTrackedMeasurementsEvent` is removed it is used in the state machine right?

Reply:
This code is now in the `loadMeasurements` command. It is not removed, just moved. 😊